notebooks/en/issues_in_text_dataset.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<p>The same dataset is available on the Hugging Face Hub - <a href="https://huggingface.co/datasets/PolyAI/banking77" target="_blank">https://huggingface.co/datasets/PolyAI/banking77</a> . </p><p>You can include an alternative way to load the data, something like this: </p><p> </p><pre class="ql-syntax" spellcheck="false">!pip -q install datasets
</pre><p> </p><pre class="ql-syntax" spellcheck="false">from datasets import load_dataset
dataset = load_dataset("PolyAI/banking77")
text_embeddings = transformer.encode(poly_dataset["train"]["text"][:1000])
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>using the hugging face dataset, there are subtle differences like the label name being in English. However, I moved to HF dataset. </p>